src/docker-compose.yml

```diff
@@ -46,8 +50,6 @@ services:
       - WEB_ROOT_URL
     build:
       context: ./products
-      args:
-        - GOPROXY_OVERRIDE
```

Comment:
Was this removed on purpose? It was added for PR #247

Reply:
I have now checked all commits since the previous merge of main into this branch and ensured they are in. I had assumed that one line you noticed previously was an anomaly because I had been editing the `.env` file, which was an incorrect assumption.